README.md

```diff
@@ -23,13 +23,13 @@ or install it from a source code checkout:
 ```go
 import "github.com/elastic/go-elasticsearch/client"
 
-c, _ = client.New(client.WithHosts([]string{"https://elasticsearch:9200"}))
+c, _ = client.New(client.WithHost([]string{"https://elasticsearch:9200"}))
```

Comment:
several problems here:
1. `WithHost`  accept `string` as param, not array of strings `[]string`
2. variable `c` should be inited like `c := ...`
3. bad practice ignoring `error`, better to check it for `nil`